dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryAPIImpl.java

```diff
@@ -574,4 +576,61 @@ public synchronized String suggestVelocityVarName(final String categoryVelVarNam
 	    }
 	}
 
+    /**
+     * Util method to check and generate (if necessary) a unique key for the Category.
+     *
+     * @return {@link Category} with a unique key.
+     */
+    private Category checkUniqueKey(Category category, User user)
+            throws DotDataException, DotSecurityException {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)